### DISCUSSION TITLE
3DTouch failing to level 19 out of 20 tries

### DISCUSSION THREAD

**Discussion by Aspyryan** [mriscoc/Ender3V2S1#1386] [Q&A]
Hi, I recently got a 3D touch, installed the cables, disconnected z-switch and installed this firmware. Now when trying to auto home / level, whenever the rod touches the plate, it retracks and stops the z-motor correctly, but it just doesn't do anything and flashes red. Why is this happening? Anything I could try? I have replaced the rod, played with the hex screw tight and untight but nothing seems to work.

And yes is normally is screwed in but I was just testing around.

https://github.com/user-attachments/assets/1ab57128-f86b-495f-978f-040b7e6b20ed

**Comment by mriscoc** [comment]
Check de white wire

**Reply by Aspyryan** [reply]
its fully seated and glued down. Should I reglue it?

**Reply by mriscoc** [reply]
Also review that the white and black wires are connected on the bltouch socket and not in the z-end stop socket of the board.

**Reply by Aspyryan** [reply]
Yea both connected to the dedicated pins in the 5pin-connector and the z-stop is unconnected
